tools/Makefile

```diff
@@ -3,10 +3,12 @@ KALDI =
 
 .PHONY: all clean
 
-all: venv chainer_patch.done kaldi.done nkf.done warp-ctc.done chainer_ctc.done sentencepiece.done kaldiio moses
+all: venv chainer_patch.done kaldi.done warp-ctc.done chainer_ctc.done kaldiio
 
 all_python: venv chainer_patch.done warp-ctc.done chainer_ctc.done
 
+extra: nkf.done sentencepiece.done mecab.done moses
```

Comment:
I like this modification, but people use the basic installation without `extra` will be stacked. We could need to think of several solutions
1. add an installation check for these tools when they are used. Kaldi is handling extra tools like that, e.g., https://github.com/kaldi-asr/kaldi/blob/master/egs/chime4/s5_1ch/local/run_beamform_6ch_track.sh#L30-L34
2. include `extra` in `all`.

Some tools (e.g., `sentencepiece`) are kind of necessary tools for us. So, I like the solution 2.